extension/arquillian-drone-saucelabs-extension/README.adoc

```diff
@@ -0,0 +1,131 @@
+= Arquillian Drone Sauce Labs extension
+This is an Arquillian extension that enables you to use the Sauce Labs as a Drone WebDriver there in your functional tests.
+
+=== How to use it:
+
+1):: First of all, you need to have this extension on your project's classpath. Eg. as a maven dependency:
+[source,xml]
+----
+    <dependency>
+        <groupId>org.jboss.arquillian.extension</groupId>
+        <artifactId>arquillian-drone-saucelabs-webdriver</artifactId>
+        <version>${version.org.jboss.arquillian.drone}</version>
+    </dependency>
+----
+
+2):: To use the SauceLabsDriver you need to specify *saucelabs* as a *browser* property:
+[source,xml]
+----
+    <extension qualifier="webdriver">
+        <property name="browser">saucelabs</property>
+        ...
+    </extension>
+----
+
+3):: You can choose if you want to specify your credentials to access your account or to provide the URL directly with the credentials included.
+
+::    a) To provide only the credentials you need to specify the *username* and *access.key* (in this case, the URL is created by the pattern: _http:// + username + : + accessKey + @ondemand.saucelabs.com:80/wd/hub_)
+[source,xml]
+----
+    <property name="username">*****</property>
+    <property name="access.key">*****</property>
+----
+
+::    b) To provide the URL directly, please use the *url* parameter:
+[source,xml]
+----
+    <property name="url">http://xxxxx:zzzzzz@ondemand.saucelabs.com:80/wd/hub</property>
+----
+
+4):: Specify there in your arquillian.xml any Sauce Labs capability that you need for testing. All the properties are passed to the WebDriver as the Capabilities. For example:
+[source,xml]
+----
+    <extension qualifier="webdriver">
+        <property name="browser">saucelabs</property>
+
+        <property name="username">*****</property>
+        <property name="access.key">*****</property>
+
+        <property name="browserName">firefox</property>
+        <property name="platform">OS X 10.9</property>
```

Comment:
From the Sauce Labs web sites. For example here are all the combinations https://saucelabs.com/platforms/ and here is configurator https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/